appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: "{build}"
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
Because git will check out text files with CRLF line endings by default. he code style checker that is part of the tests here will fail since the line endings won't be LF.